bibtex-actions-file.el

```diff
@@ -50,6 +50,12 @@ If you use 'org-roam' and 'org-roam-bibtex, you should use
   :group 'bibtex-actions
   :type '(function))
 
+(defcustom bibtex-actions-file-parser-function
+  'bibtex-actions-file-parser-default
+  "Function to parse file field"
+  :group 'bibtex-actions
+  :type '(function))
```

Comment:
Two questions, maybe either of which we need to answer for sure now:

1. should there be an explicit choice here?
2. do we think there might be need to allow more than one function?

Reply:
Basic support (just a path) seems to be a good default. If more than one parser is required we can support a list of functions to call or provide the user a way to create another custom function doing the job (`seq-mapcat (lambda (fun) (apply fun ...)))

Reply:
I think a single functions is a good thing here. Especially since the functions takes arguments and is not just for side effects. I think if we allow multiple functions, the intent would be try a bunch of parsers until one return non nil? That can be done pretty easily directly too. 

I think there should be some explanations of what the arguments for the function are in the docstring of the `defcustom`.